src/root/edit-jobset.tt

```diff
@@ -20,18 +20,9 @@
     <td id="[% baseName %]">
       [% IF createFromEval %]
         [% value = (input.uri or input.value); IF input.revision; value = value _ " " _ input.revision; END;
```

Comment:
Can this entire branch be removed? I can't tell if it's used or not, but it doesn't appear so to me...